docs/examples/viz_animated_surfaces.py

```diff
@@ -0,0 +1,201 @@
+eq3 = "(np.sin(np.pi*2*x-np.sin(1.8*time))*np.cos(np.pi*2*y+np.cos(1.8*time)))\
+      *0.48"
+eq4 = "np.cos(24*np.sqrt(x**2 + y**2) - 2*time)*0.18"
+equations = [eq1, eq2, eq3, eq4]
+
+###############################################################################
+# List of colormaps to be used for the various functions.
+cmap_names = ['hot', 'plasma', 'viridis', 'ocean']
+
+###############################################################################
+# Creating a list of wave actors.
+waves = []
+for i in range(4):
+    waves.append(wave(x, y, equation=equations[i],
+                 colormap_name=cmap_names[i]))
+actors = waves
+
+
+###############################################################################
+# Creating a scene object and configuring the camera's position
+
+scene = window.Scene()
+scene.set_camera(position=(4.45, -21, 12), focal_point=(4.45, 0.0, 0.0),
+                 view_up=(0.0, 0.0, 1.0))
+showm = window.ShowManager(scene, size=(600, 600))
+
+###############################################################################
+# Creating an axes actor(for reference) and a grid to store multiple functions.
+# Adding them both to the scene.
+
+# To store the function names
+text = []
+for i in range(4):
+    t_actor = actor.label('Function ' + str(i + 1), pos=(0, 0, 0), scale=(0.17, 0.2, 0.2))
+    t_actor.SetCamera(scene.camera())
+    text.append(t_actor)
+
+# grid
+grid_ui = ui.GridUI(actors=actors, captions=text,
+                    caption_offset=(-0.7, -2.5, 0), dim=(1, 4),
+                    cell_padding=2,
+                    aspect_ratio=1,
+                    rotation_axis=(0, 1, 0))
+showm.scene.add(grid_ui)
+showm.scene.add(actor.axes())
+
+
+###############################################################################
+# Initializing text box to print the title of the animation
+tb = ui.TextBlock2D(bold=True, position=(200, 60))
+tb.message = "Animated 2D functions"
+scene.add(tb)
+
+###############################################################################
+# Initializing showm and counter
+showm.initialize()
+counter = itertools.count()
+
+###############################################################################
+# end is used to decide when to end the animation
+end = 200
+
+
+###############################################################################
+# Coordinates to be plotted are changed everytime timer_callback is called by
+# using the update_coordinates function. The 2D functions are rendered here
+
+def timer_callback(_obj, _event):
+    global xyz, time
+    time += dt
+    cnt = next(counter)
+
+    # updating the colors and vertices of the triangles used to form the waves
+    for wave in waves:
+        xyz, colors = update_coordinates(x, y, equation=wave.equation,
+                                         cmap_name=wave.cmap_name)
+        wave.GetMapper().GetInput().GetPointData().\
+            SetScalars(utils.numpy_to_vtk_colors(255*colors))
```

Comment:
@SunTzunami Create an utility function to do this job.

Reply:
I am talking about ``.GetMapper().GetInput().GetPointData()``

Reply:
Done